### ISSUE TITLE
update specs failing with TypeError

### ISSUE THREAD

**Issue by ScottMaclure** [aws-cloudformation/cfn-python-lint#670]
Running from WSL/Win10:

```bash
$ python3 --version
Python 3.5.2

$ pip3 list | grep cfn-lint
cfn-lint                 0.14.1

$ cfn-lint --version
cfn-lint 0.14.1

$ sudo cfn-lint -u                                                                                                                                                                                                 Traceback (most recent call last):
  File "/usr/local/bin/cfn-lint", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/cfnlint/__main__.py", line 28, in main
    (args, filenames, formatter) = cfnlint.core.get_args_filenames(sys.argv[1:])
  File "/usr/local/lib/python3.5/dist-packages/cfnlint/core.py", line 120, in get_args_filenames
    cfnlint.maintenance.update_resource_specs()
  File "/usr/local/lib/python3.5/dist-packages/cfnlint/maintenance.py", line 63, in update_resource_specs
    content = json.loads(req.content)
  File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
```

**Comment by kddejong** [comment]
Going to try to setup a way to test this but not my normal environment so I apologize for any slowness here.

**Comment by kddejong** [comment]
Same issue on linux.  Looks like we don't have good testing for this on Python 3.5 (and I'm guessing Python 3.4).  Much easier for me to replicate now so should have a fix out there shortly.

**Comment by kddejong** [comment]
Should be fixed in #697.  Will be in the next of release.
